docs/NNNN-n-tuple-omit-prefix-storage-layout.md

```diff
@@ -0,0 +1,209 @@
+# OCFL Community Extension NNNN: N Tuple Omit Prefix Storage Layout
+
+  * **Extension Name:** NNNN-n-tuple-omit-prefix-storage-layout
+  * **Authors:** Michael Vandermillen and Andrew Woods
+  * **Minimum OCFL Version:** 1.0
+  * **OCFL Community Extensions Version:** 1.0
+  * **Obsoletes:** n/a
+  * **Obsoleted by:** n/a
+
+## Overview
+
+This storage root extension describes an OCFL storage layout combining a pairtree-like root directory structure derived from prefix-omitted object identifiers, followed by the prefix-omitted object identifier themselves. 
+The OCFL object identifiers are expected to contain prefixes which are removed in the mapping to directory names. 
+The OCFL object identifier prefix is defined as all characters before and including a configurable delimiter.
+Where the prefix-omitted identifier length is less than tuple size * number of tuples, the remaining object id (prefix omitted) is left or right-side, zero-padded (configurable, left default), or not padded (none), and optionally reversed (default false).
+The object id is then divided into N n-tuple segments, and used to create nested paths under the OCFL storage root, followed by the prefix-omitted object id directory.  
+
+This layout combines the advantages of 0006-flat-omit-prefix-storage-layout (directory name transparency) and the 0004-hashed-n-tuple-storage-layout (enhanced file system/bucket performance). 
+
+The limitations of this layout are filesystem dependent, but are generally as follows:
+
+* The size of object identifiers, minus the length of the prefix, cannot exceed the maximum allowed directory name size
+  (eg. 255 characters)
+* Object identifiers cannot include characters that are illegal in directory names
+
+## Parameters
+
+### Summary
+
+* **Name:** `delimiter`
+  * **Description:** The case-insensitive, delimiter marking the end of the OCFL object identifier prefix; MUST consist
+    of a character string of length one or greater. If the delimiter is found multiple times in the OCFL object
+    identifier, its last occurence (right-most) will be used to select the termination of the prefix.
+  * **Type:** string
+  * **Constraints:** Must not be empty
+  * **Default:** :
+* **Name**: `tupleSize`
+  * **Description:** Indicates the segment size (in characters) to split the
```

Comment:
The other n-tuple extensions create tuples out of a hex encoded digest, which must be ASCII. However, this extension operates on the object id itself, which is UTF-8. Because of this, I think greater care needs to be taken describing how the id is segmented, if only to draw attention to the problem so that it is not overlooked in implementations. My assumption is that the intent is to split on grapheme clusters?

Reply:
Yes, I added an explanation under Procedure